### PR TITLE
fix help message when white noise issue occurs

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
When ci lint or `make lint` command checks white noise, the suggestion message should suggest using `git status` or `git diff` to find the reason, but not using `make lint` command. Or, developers or who first encounters whitenoise,  don't know how to fix this problem.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/lib/lint.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ kubeedge::lint::check() {
     fi
 
     [[ $(git diff --name-only) ]] && {
-      echo "Some files have white noise issue, please run \`make lint\` to slove this issue"
+      echo "Some files have white noise issue, please run \`git status\` to find whitenoise and solve this issue"
```
Comment (fisherxu) [inline hack/lib/lint.sh:64]:
```suggestion
      echo "Some files have white noise issue, please run \`make lint\` to solve and \`git status\` to find and fix this issue"
```

Comment (fisherxu) [reply hack/lib/lint.sh:64]: If users find this issue in the ci, they also need to run make lint in local env.

Comment (gy95) [reply hack/lib/lint.sh:64]: ok

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**